Examples/FireBaseUI/FireBaseUIViewController.swift

```diff
@@ -195,4 +236,27 @@ internal final class FireBaseUIViewController: ViewController {
       }
     }
   }
+
+  private func fetchUserDocument() {
+    guard let user = Auth.auth().currentUser else {
+      print("No current user, can't fetch document...")
+      return
+    }
+
+    Task {
+      do {
+        let firestore = Firestore.firestore()
+        let document = firestore
+          .collection("users")
+          .document(user.uid)
```

Comment:
Continuations should be indented an extra level:

```suggestion
            .collection("users")
            .document(user.uid)
```

Reply:
Any way to make the editor do this automatically? I'm happy to fix these manually, but we will need a better solution for other people.